test/testdata.py

```diff
@@ -20,8 +20,7 @@ def loadData(self) -> None:
         self.temperature = file["temperature"]
 
     def update(self) -> None:
```

Comment:
I'm curious why update needs to be separate from get(), I would think these would always be called back to back?